crates/walrus-service/src/client/cli/args.rs

```diff
@@ -1173,50 +1164,56 @@ pub struct QuiltPatchQuery {
 impl QuiltPatchQuery {
     /// Returns a QuiltPatchSelector from the command line arguments.
     pub fn to_selector(&self) -> Result<QuiltPatchSelector> {
```

Comment:
Suggestion: Change this to take an owned reference, then we can avoid the clones.
```suggestion
    pub fn into_selector(self) -> Result<QuiltPatchSelector> {
```
This may require doing a clone of the whole query, but it exposes the fact that we do in fact clone everything to the caller.